Problems/etc/lamda_use.py

```diff
@@ -0,0 +1,17 @@
+# lamda �Ű����� : ���ϰ�
+# map = �Լ�, list ��Ҹ� �Լ� �ְ� ��ȯ�� ����� ��ο� ����Ʈ ����
+# filter = �Լ�, list ��Ҹ� �Լ� �ְ� ��ȯ�� ���� True�� ��쿡�� ��ο� ����Ʈ ��
+
+# ����(1)
+power = lambda x:x*x
+under_3 = lambda x:x<3
+
+list_input_a = [1,2,3,4,5]
+output_a = map(power, list_input_a)
+output_b = filter(under_3, list_input_a)
```

Comment:
```suggestion
output_b = filter(lambda x:x<3, list_input_a)
```
위와 같은 이유로, 주로 사용하는 형태는 suggestion과 같이 한번 쓰고 버리는(따로 함수명명 안하고) 형태인 경우에만 제한적으로 사용.